velox/functions/sparksql/tests/MightContainTest.cpp

```diff
@@ -25,17 +25,34 @@ namespace {
 class MightContainTest : public SparkFunctionBaseTest {
  protected:
   void testMightContain(
-      const VectorPtr& bloom,
+      const std::optional<std::string>& serialized,
```

Comment:
This is VectorFunction, so we need to use input Vector, previously I use StringView as input, the final version changes it to vector

Reply:
I think we don't need to pass a Vector here. Previously the first input argument is a vector of string, but now the serialized string will be added to the argument list as a single constant.